### ISSUE TITLE
Black Screen - Stream - macOS Monterey

### ISSUE THREAD

**Issue by DevCode01** [unknownskl/xbox-xcloud-client#641]
Hi !
I cannot use Greenlight because I've a black screen without any error when I want to stream my xbox series s on my Mac.
Cloud gaming (xbox pass) is working great.
I've exactly the same problem with my iPhone on IOS 16 and XBOX official application.

I think it's causes by Apple but i didn't see anyone else complaining...
If someone has a solution, I'm interested

**Comment by defatlas** [comment]
im having same issue but on steam deck in gamemode only and when i log in it flashes the on screen keyboard for a second

**Comment by defatlas** [comment]
> im having same issue but on steam deck in gamemode only and when i log in it flashes the on screen keyboard for a second

nevermind about it only not working in gamemode maybe it has to do with being on different network

**Comment by DevCode01** [comment]
Yes and on my own network it's working but when I went to play outside (4G-5G/Other wifi) it's not working

**Comment by burningyouth** [comment]
Any solutions? Can't see the stream even in my home network.

**Comment by kikiwora** [comment]
Hm. I had the black screen issue on Monterey too.
The reason, as it seems, that app is unable to wake up my console.
If turned on manually, streaming is possible, however it is slow for some reason, despite using 5GHz Wi-Fi via node located in the same room.

Also, the sound is garbaged and latency is significant.

Works on my iPhone using first-party Xbox app as expected though.

**Comment by toby1991** [comment]
Use your first-party Xbox app's streaming login, then change the screen resolution from 720p to 1080p, save, quit streaming.

Then third-party streaming will works
